packages/CopyInput/src/CopyInput.js

```diff
@@ -85,7 +90,7 @@ function CopyInput(props) {
         <Popover.Tip />
       </Popover>
       {isClickedTooltipOpen && (
-        <Toast kind={Toast.types.kind.VISUALLY_HIDDEN} aria-hidden>
+        <Toast kind={Toast.types.kind.VISUALLY_HIDDEN} isPolite>
```

Comment:
Good call with the `isPolite`, but I think it's worth keeping the `aria-hidden` so that screen reader users don't navigate to this element and read it out of context.